sn_client/src/api.rs

```diff
@@ -106,14 +104,8 @@ impl Client {
         let root_dir = std::env::temp_dir();
         trace!("Starting Kad swarm in client mode..{root_dir:?}.");
 
-        #[cfg(not(feature = "open-metrics"))]
         let network_builder = NetworkBuilder::new(Keypair::generate_ed25519(), local, root_dir);
 
-        #[cfg(feature = "open-metrics")]
-        let mut network_builder = NetworkBuilder::new(Keypair::generate_ed25519(), local, root_dir);
-        #[cfg(feature = "open-metrics")]
-        network_builder.metrics_registry(Some(Registry::default()));
-
```

Comment:
Unused code. The port has to be set to enable the metrics server, and it is not done for Client atm.